src/types.ts

```diff
@@ -0,0 +1,52 @@
+interface ValidateTextOptionsBase {
+	medium?: 'all' | 'braille' | 'embossed' | 'handheld' | 'print' | 'projection' | 'screen' | 'speech' | 'tty' | 'tv';
+}
+
+type ValidateTextOptionsWithBaseWarning = ValidateTextOptionsBase & {
```

Comment:
Instead of "with base warning," what about the phrase "without warnings" instead?

```suggestion
type ValidateTextOptionsWithoutWarnings = ValidateTextOptionsBase & {
```